workspace.go

```diff
@@ -481,6 +485,9 @@ type WorkspaceUpdateOptions struct {
 	// from another workspace.
 	AutoApplyRunTrigger *bool `jsonapi:"attr,auto-apply-run-trigger,omitempty"`
 
+	// Optional: The time after which an automatic destroy run will be queued
+	AutoDestroyAt *Unsettable[time.Time] `jsonapi:"attr,auto_destroy_at,omitempty"`
```

Comment:
this not matching the shape of `AutoDestroyAt` values in the other Workspace structs doesn't feel great 😕

Reply:
this is being addressed upstream in [our fork of jsonapi](https://github.com/hashicorp/jsonapi/pull/21).